src/redux/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+import categoryReducer from './categories/categories';
+
+const reducer = combineReducers({
+  booksReducer,
+  category: categoryReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger),
+);
+
+export default store;
```

Comment:
- [ ] Good job installing the `@reduxjs/toolkit` package in the `package.json` file. However, you need to import `configureStore` from it to configure your store. Kindly refactor accordingly. See example:

  ```
  import { combineReducers } from 'redux';
  import { configureStore } from '@reduxjs/toolkit';
  import booksReducer from './books/books';
  import categoryReducer from './categories/categories';
  
  const reducer = combineReducers({
    books: booksReducer,
    categories: categoryReducer,
  });
  
  const store = configureStore({
    reducer,
  });
  
  export default store;
  ```